RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -4,6 +4,8 @@
 
 fCPerMIP_v10 = cms.vdouble(2.06,3.43,5.15) #120um, 200um, 300um
 
+fCPerMIP_v16 = fCPerMIP_v10
```

Comment:
I do not think there's a need to version the `fCPerMIP`. Would it make sense to just reuse `fCPerMIP_v10`?
Or, even better, remove the versioning from the variable's name completely?
I believe that could also imply removing the last four lines you have added, right?

Reply:
@rovere The default values are with the MPV, while the ones with the version are with the mean. Isn't it better to remove the versioning and introduce two variables (`fCPerMIP_mean`, `fCPerMIP_mpv`) and in each modifier use the appropriate one?

Reply:
@apsallid absolutely yes, that would be the optimal choice indeed!